modules/yaml-schema.json

```diff
@@ -159,15 +157,27 @@
                                 "items": {
                                     "type": "string"
                                 },
-                                "uniqueItems": true
+                                "uniqueItems": true,
+                                "minItems": 2,
```

Comment:
Maybe a silly question, but does this only apply when it's of type "array", or also when it's of type "string" ( and in which case what does that mean?).

Reply:
tested it and this option is ignored for strings.